src/libprojectM/projectM.cpp

```diff
@@ -70,29 +71,22 @@ projectM::~projectM()
 {
     void *status;
 
-    #ifdef USE_THREADS
+#ifdef USE_THREADS
     std::cout << "[projectM] thread ";
     printf("c");
```

Comment:
whats with the printfs

Reply:
Remove imo

Reply:
I mean are they printed? Is this code run?

Reply:
There is uncommented debug code in QPulseAudioThread.cpp and TextureManager.cpp as well.  Might be nice to have a runtime DEBUG flag as well as a compile time flag.  When I have a few minutes I could try to do a pass.